libvncclient/tls_openssl.c

```diff
@@ -18,6 +18,8 @@
  *  USA.
  */
 
+#include <stdio.h>
```

Comment:
can you please add it under the errno.h include?

Reply:
ah and this probably also holds true for tls_gnutls.c?

Reply:
For sure!

Reply:
Done.